core/controllers/editor.py

```diff
@@ -79,7 +83,10 @@ class ExplorationHandler(EditorHandler):
 
     @acl_decorators.can_play_exploration
     def get(self, exploration_id):
-        """Gets the data for the exploration overview page."""
+        """Handles GET requests.
+
+        Args:
+            exploration_id: Runs the data for the exploration overview page."""
```

Comment:
Move the `"""` to the next line which is the format for multiline docstrings.
Also, change text to "Returns the data for the exploration editor page."
Arg type is also missing? (It should be `exploration_id: str. Returns the ...`)